content/authors/masese-gavin/index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Masese Gavin
+type: authors
+github: https://github.com/masese911
+---
+ I’m currently learning computer science. I’m looking to collaborate on software development. I play games whenever i am free.
```

Comment:
Look at other author descriptions. This is an excellent opportunity for people to recognize you. Make sure you add all the required details.